vsts/pipelines/nightly.yml

```diff
@@ -20,8 +20,6 @@ jobs:
       echo "##vso[task.setvariable variable=BuildBuildImages;]true"
       echo "##vso[task.setvariable variable=TestBuildImages;]true"
       echo "##vso[task.setvariable variable=PushBuildImages;]true"
-      echo "##vso[task.setvariable variable=PushRuntimeImages;]false"
-      echo "##vso[task.setvariable variable=EmbedBuildContextInImages;]true"
```

Comment:
When build context is embedded, it causes cache invalidation (due to changes in git commit and build number) and hence causing the build to be slow. Since we `throw away` the images built in Nightly build, we can ignore it. We need to  however embed in CI builds.